config/source.json

```diff
@@ -37,9 +37,12 @@
         }
     },
     "attr": {
-        "type": "git",
-        "rev": "v2.5.2",
-        "url": "https://git.savannah.nongnu.org/git/attr.git",
+        "alt": {
+            "type": "url",
+            "url": "https://mirror.souseiseki.middlendian.com/nongnu/attr/attr-2.5.2.tar.gz"
```

Comment:
yes, this is an official mirror linked to from savannahs download page

Reply:
Move original downloads to `alt` will be good. We dont' have `alt` download option yet.

Reply:
I know, preparing for the future. Their downloads page seems more stable than the git. I think it's fine the way it is.